File: Python For Everyone, Horstmann/Chapter5-Functions/P5.10.py
# Write a function
#     def readFloat(prompt)
# that displays the prompt string, followed by a space, reads a floating-point number
# in, and returns it. Here is a typical usage:
#     salary = readFloat("Please enter your salary:")
#     percentageRaise = readFloat("What percentage raise would you like?")

# FUNCTIONS
def readFloat(prompt):
    return input(prompt + " ")

def main():
    salary = readFloat("Please enter your salary:")
    percentageRaise = readFloat("What percentage raise would you like?")

    # tests
    print("Salary entered?", salary)
    print("Percentage raise entered?", percentageRaise)

# PROGRAM RUN
main()

File: Java-for-Everyone-Late-Objects-Horstmann/Chapter-5-Methods/P5_10.java
/*Write a method
public static double readDouble(String prompt)
that displays the prompt string, followed by a space, reads a floating-point number
in, and returns it. Here is a typical usage:
salary = readDouble("Please enter your salary:");
percentageRaise = readDouble("What percentage raise would you like?");*/

import java.util.Scanner;

public class P5_10 {
    public static double readDouble(String prompt) {
        Scanner input = new Scanner(System.in);
        System.out.print(prompt + " ");
        double input_number = input.nextDouble();
        input.close();
        return input_number;
    }
}
